### ISSUE TITLE
doesn't work after v0.10.2

### ISSUE THREAD

**Issue by kurokirasama** [mmabrouk/chatgpt-wrapper#302]
**Describe the bug**
running anything gives this error (`chatgpt, chatgpt help, chatgpt config, etc):
```
WARNING: backend 'chatgpt-api' has been renamed to 'api', and support for the old name will be removed in a future release. Please update your config file to 
use 'api' for the 'backend:' setting.

SchemaUpdater - ERROR - Error during schema update process: Path doesn't exist: '/home/kira/.local/lib/python3.8/site-packages/chatgpt_wrapper/backends/api/schema/alembic'.  Please use the 'init' command to create a new scripts folder.
SchemaUpdater - ERROR - Stack trace: Traceback (most recent call last):
  File "/home/kira/.local/lib/python3.8/site-packages/chatgpt_wrapper/backends/api/schema/updater.py", line 84, in update_schema
    latest_version = self.get_latest_version()
  File "/home/kira/.local/lib/python3.8/site-packages/chatgpt_wrapper/backends/api/schema/updater.py", line 58, in get_latest_version
    script = ScriptDirectory.from_config(self.alembic_cfg)
  File "/home/kira/.local/lib/python3.8/site-packages/alembic/script/base.py", line 216, in from_config
    return ScriptDirectory(
  File "/home/kira/.local/lib/python3.8/site-packages/alembic/script/base.py", line 97, in __init__
    raise util.CommandError(
alembic.util.exc.CommandError: Path doesn't exist: '/home/kira/.local/lib/python3.8/site-packages/chatgpt_wrapper/backends/api/schema/alembic'.  Please use the 'init' command to create a new scripts folder.

An error occurred during the schema update process. Please check the logs
```

**To Reproduce**
already described

**Expected behavior**
it should open chatgpt

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] ubuntu 20.04
 - Browser [e.g. chrome, safari] chrome
 - Version [e.g. 22] v0.10.2

**Smartphone (please complete the following information):**

**Additional context**
log attached
[chatgpt.log](https://github.com/mmabrouk/chatgpt-wrapper/files/11474385/chatgpt.log)

**Comment by vivek09thakur** [comment]
have you find any solution about it?

**Comment by thehunmonkgroup** [comment]
Clearly the Python packaging system and I are not friends...

I believe the problems are now fixed in 0.10.3.
